en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-soap-api-passthrough.md

```diff
@@ -0,0 +1,311 @@
+# Deploying a SOAP API(Pass-through)  in Choreo Connect
+
+!!! attention "Update Level 90"
```

Comment:
have to update this accordingly, once the update is pushed

Reply:
Updated the level as 4 https://github.com/wso2/docs-apim/pull/6080/commits/a68a28d326d718c2cc992038177f7d9964b190c3